views/userManagementBadge.njk

```diff
@@ -0,0 +1,133 @@
+    <tr>
+        <th>Name</th>
+        <td>{{user.first}} {{user.last}}</td>
+    </tr>
+    <tr>
+        <th>Badge ID</th>
+        <td>{{user.badge}}</td>
+    </tr>
+    <tr>
+        <th>Email</th>
+        <td>{{user.email}}</td>
+    </tr>
+    <tr>
+        <th>Phone</th>
+        <td>{{user.phone}}</td>
+    </tr>
+    <tr>
+        <th>Signature</th>
+        <td>{{user.signature}}</td>
+    </tr>
+    <tr>
+        <th>Parent/Guardian signature</th>
+        <td>{{user.ecSignature}}</td>
+    </tr>
+    <tr>
+        <th>Emergency contact name</th>
+        <td>{{user.ecName}}</td>
+    </tr>
+    <tr>
+        <th>Emergency relationship</th>
+        <td>{{user.ecRel}}</td>
+    </tr>
+    <tr>
+        <th>Emergency phone number</th>
+        <td>{{user.ecPhone}}</td>
+    </tr>
+    <tr>
+        <th>Updated at: </th>
+        <td>{{user.updatedAt}}</td>
+    </tr>
+    <tr>
+        <th>Created at: </th>
+        <td>{{user.createdAt}}</td>
+    </tr>
+    <tr>
+        <th></th>
+        {% if user.confirmation %}
+            <td><button class="btn btn-primary"  onclick= "delUser(this)" data-badge={{user.badge}}>Delete</button></td>
+        {% else %}
+            <td><button class="btn btn-primary"  onclick= "valUser(this)" data-badge={{user.badge}}>Validate</button></td>
+        {% endif %}
+    </tr>
+
+  </table>
+
+{# Station listing #}
+
+  <table class="table table-bordered" style="width:55%; display:inline-block; align:right; vertical-align: top">
+    <thead>
+        <tr>
+            <th scope="col" nowrap="nowrap">Station ID</th>
+            <th scope="col">Name</th>
+            <th scope="col">Description</th>
+            <th scope="col">Status</th>
+            <th scope="col">Action</th>
+        </tr>
+    </thead>
+        {% for station in allStations %}
+        <tr>
+            <td>{{station.sId}}</td>
+            <td>{{station.name}}</td>
+            <td>{{station.description}}</td>
+            {% if station.trained %}
+                <td>Trained</td>
+                <td><button type="button" class="btn btn-primary btn-md"  onclick= "revokePriv(this)" data-sID = {{station.sId}} data-badge={{user.badge}}>Revoke</button></td>
+            {% else %}
+                <td>Not Trained</td>
+                <td><button type="button" class="btn btn-primary btn-md"  onclick= "grantPriv(this)" data-sID = {{station.sId}} data-badge={{user.badge}}>Grant Access</button></td>
+            {% endif %}
+        </tr>
+        {% endfor %}  
+  </table>
+</div>
+<script type="text/javascript">
+
+        function delUser(user) {
+            var badge  = user.getAttribute("data-badge")     
+            window.location.replace("/userManagement/deleteUser/" + badge)
+        }
+
+        function valUser(user) {
+            var badge  = user.getAttribute("data-badge")
+            window.location.replace("/userManagement/modifyUser/" + badge)
```

Comment:
change this route to be /userManagement/confirmUser/ instead of  /userManagement/modifyUser/ 
to be worked with my route